server/views/pages/bookVisit/additionalSupport.njk

```diff
@@ -22,12 +22,17 @@
       <form action="{{ paths.BOOK_VISIT.ADDITIONAL_SUPPORT }}" method="POST" novalidate>
         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
         {% set supportDetailsHtml %}
-          {{ govukInput({
+          {{ govukCharacterCount({
             id: "additionalSupport",
             name: "additionalSupport",
+            maxlength: 200,
             type: "text",
-            classes: "govuk-!-width-one-half",
+            classes: "govuk-!-width-two-thirds",
```

Comment:
Do we need to add any `classes` here? Worth trying without as the parent column is already two-thirds...